public/css/style.scss

```
@@ -525,8 +550,13 @@ footer {
 /* header */
 .jumbotron {
 	margin: 0;
-	h1, h2, h3, h4, h5, p {
-		font-family: 'Titillium Web', sans-serif;
+	h1,
+	h2,
+	h3,
+	h4,
+	h5,
+	p {

```

This shortens the resulting `.css` by combining otherwise duplicated selectors.

```suggestion
	:where(h1, h2, h3, h4, h5, p) {
```